test-harness/src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -2330,13 +2199,13 @@ public boolean isLoggable(LogRecord record) {
         // this also prevents tests from falsely advertising Hudson
         DNSMultiCast.disabled = true;
 
-        if (Functions.isGlibcSupported()) {
-            try {
-                GNUCLibrary.LIBC.unsetenv("MAVEN_OPTS");
-                GNUCLibrary.LIBC.unsetenv("MAVEN_DEBUG_OPTS");
-            } catch (Exception e) {
-                LOGGER.log(Level.WARNING,"Failed to cancel out MAVEN_OPTS",e);
-            }
+        try {
+            GNUCLibrary.LIBC.unsetenv("MAVEN_OPTS");
+            GNUCLibrary.LIBC.unsetenv("MAVEN_DEBUG_OPTS");
+        } catch (LinkageError x) {
+            // skip; TODO 1.630+ can use Functions.isGlibcSupported
```

Comment:
`Functions.isGlibcSupported` did not print anything. Just following suit. No opinion otherwise.